src/tests/ftest/osa/online_drain.py

```diff
@@ -186,3 +192,17 @@ def test_osa_online_drain_mdtest(self):
         """
         self.log.info("Online Drain : With Mdtest")
         self.run_online_drain_test(1, app_name="mdtest")
+
+    def test_osa_online_drain_with_multiple_ranks(self):
+        """Test ID: DAOS-4753.
+
+        Test Description: Drain multiple ranks at the same time.
+
+        :avocado: tags=all,daily_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=osa,osa_drain,online_drain
+        :avocado: tags=OSAOnlineDrain,test_osa_online_drain_with_multiple_ranks
+        """
+        self.log.info("Online Drain : Test with multiple ranks")
+        multiple_ranks = self.params.get("rank_list", '/run/multiple_ranks/*')
+        self.run_online_drain_test(1, multiple_ranks=multiple_ranks)
```

Comment:
With num_ranks, we'd call it:
```suggestion
        self.run_online_drain_test(1, num_ranks=2)
```